doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -0,0 +1,72 @@
+.. _rendering-documentation:
+
+===================================
+Rendering Documentation with Sphinx
+===================================
+
+SciPy docstrings are rendered to html using `Sphinx`_. Writing
+docstrings is covered in the `numpydoc docstring guide`_; this document
+explains how to check that docstrings render properly.
+
+*For a video walkthrough, please see* \ `Rendering SciPy Documentation
+with Sphinx`_ \ *.*
+
+.. _rendering-documentation-locally:
+
+Rendering Documentation Locally
+-------------------------------
+
+To render the documentation on your own machine (macOS or Linux):
+
+#. Install `Sphinx`_ or ensure that your installation is up to date.
```

Comment:
Ensuring that the user is in the correct development environment and providing the "conda update sphinx" command would make installing Sphinx very simple and easy. This step is also presented in the video walkthrough, as opposed to installing Sphinx through the linked website.